probcal/enums.py

```diff
@@ -32,6 +33,9 @@ class DatasetType(Enum):
 class ImageDatasetName(Enum):
     MNIST = "mnist"
     COCO_PEOPLE = "coco_people"
+    CIFAR10 = 'cifar_10'
+    CIFAR100 = 'cifar_100'
```

Comment:
Let's leave this out unless we have those datasets implemented

Reply:
Fixed.